.lintstagedrc.mjs

```diff
@@ -0,0 +1,3 @@
+export default {
+  '**/*.{mjs,ts,tsx,md,mdx,css,sass,scss}': 'npm run lint:fix',
```

Comment:
I am thinking of changing this to `npm run format` so it covers lint and prettier fixes

Reply:
I think that definitely makes more sense to run `npx turbo format`